pkg/protocol/evictionplugin/v1alpha1/api.proto

```diff
@@ -95,11 +95,16 @@ message GetEvictPodsResponse {
     Condition condition = 2;
 }
 
+message GetTokenResponse {
+    string token = 1;
+}
+
 message DeletionOptions {
     int64 grace_period_seconds = 1;
 }
 
 service EvictionPlugin {
```

Comment:
will this be compatibility to previous plugins when we add this?

Reply:
Yes. I will implement this method for all existing out-of-tree plugins in other pull request. Besides this, the Eviction Manager won't block the registration process unless we turn on the strictly validate switch. We won't turn on the switch until we make sure all running plugins implement this methods.